src/surge-fx/SurgeFXEditor.cpp

```diff
@@ -470,6 +470,11 @@ void SurgefxAudioProcessorEditor::makeMenu()
             auto t = -1;
             c->QueryIntAttribute("i", &t);
             men.fxtype = t;
+            if (t == 28) // skip the "Audio In" effect
+            {
```

Comment:
this 28 (vs fx_audioin or whatever) is to avoid linking the effects.h / storage header? But we should be able to link that in a cpp file this high up.